src/group.py

```diff
@@ -46,22 +51,22 @@ def path(self):
 
         while x != self.end_x or y != self.end_y:
             moves += 1
-            if x < len(self.parent.cells) - 1 and self.parent.cells[x + 1][y].can_move(self):
+            if x < len(self.parent.cells) - 1:# and (self.parent.cells[x + 1][y].can_move(self) or self.has_boat):
```

Comment:
Why is the `can_move` check/having a boat check commented out here/in this function?

Reply:
I doesn't yet work - does boat checking incorrectly